### DISCUSSION TITLE
does DMS support SQL Server 2005

### DISCUSSION THREAD

**Discussion by dianbaobao** [Mimetis/Dotmim.Sync#634] [Q&A] [answered]
Hello Mr.Pertus，does DMS support SQL Server 2005？I wanna sync databases from 2005 to 2019

**Comment by Mimetis** [comment, marked as answer]
Never tested.
If you are trying, do not use the change tracking provider.

**Reply by dianbaobao** [reply]
OK, I'll try it, thanks
